augur/filter.py

```diff
@@ -1466,13 +1466,13 @@ def run(args):
         # Always write out strains that are force-included. Additionally, if
         # we are not grouping, write out metadata and strains that passed
         # filters so far.
-        strains_to_write = distinct_sequences_to_include
+        force_included_strains = distinct_sequences_to_include
```

Comment:
This renaming confused me a bit. I think `strains_to_write` describes exactly what we are doing with this set of strains, since it is not referenced anywhere outside of the outputs. 

If anything, I think `distinct_sequences_to_include` should be renamed to `force_included_strains`. Then it's clearer what is happening in the change in afa9ef8.

Reply:
There are a lot of similar variables here, but I think `strains_to_write` is too generic in the scope of `run()`. There are other strains being written elsewhere in `run()`, and without the comment above I wouldn't know where to find the force-inclusion logic. Maybe:

- `strains_to_write ` -> `force_included_strains_to_write`
- `distinct_sequences_to_include` -> `distinct_force_included_strains`

Also noting that there are many other variables that could use better naming, but those are probably beyond the scope of this PR.

Reply:
Ah, that makes sense to me!